packages/ingest/sftp.js

```diff
@@ -194,7 +195,13 @@ module.exports.sftpMixin = (superclass) => class extends superclass {
     const pass = new PassThrough();
     readable.pipe(pass);
 
-    const params = { Bucket: bucket, Key: key, Body: pass };
+    const params = {
+      Bucket: bucket,
+      Key: key,
+      Body: pass,
+      ContentType: mime.lookup(key) || null
```

Comment:
Since `sftp` doesn't really have a `mime-type`, try to infer. If we can't infer, just let aws set the default `ContentType` value.